zfmt/ast.go

```diff
@@ -518,6 +518,21 @@ func (c *canon) proc(p ast.Op) {
 			c.space()
 			c.exprs(p.Args)
 		}
+	case *ast.Load:
+		c.next()
+		c.write("load %s", zson.QuotedString([]byte(p.Pool)))
+		if p.Branch != "" {
+			c.write(p.Branch)
```

Comment:
I wasn't thinking straight when I suggested changing this to `c.write(p.Branch)`! We definitely want the "@" in there.
```suggestion
			c.write("@%s", p.Branch)
```